scss/_layout.scss

```diff
@@ -6,4 +6,5 @@
 #{$ubtn-namespace}-block,
 #{$ubtn-namespace}-stacked {
   display: block;
-}
\ No newline at end of file
+  width: 100%;
```

Comment:
`box-sizing: border-box;`
is already define in _base.scss , I think it is not necessary to re-define it in the -block class.
what do you think ?